e2e/e2etests/test_solana_withdraw_and_call.go

```diff
@@ -0,0 +1,113 @@
+package e2etests
+
+import (
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/gagliardetto/solana-go"
+	"github.com/near/borsh-go"
+	"github.com/stretchr/testify/require"
+
+	"github.com/zeta-chain/node/e2e/runner"
+	"github.com/zeta-chain/node/e2e/utils"
+	solanacontract "github.com/zeta-chain/node/pkg/contracts/solana"
+	crosschaintypes "github.com/zeta-chain/node/x/crosschain/types"
+)
+
+// TestSolanaWithdrawAndCall executes withdrawAndCall on zevm and calls connected program on solana
+// message and zevm sender are stored in connected program pda, and withdrawn lamports are stored
+// in connected program pda and account provided in remaining accounts to demonstrate that lamports
+// can be moved to accounts in connected program as well as gateway program
+func TestSolanaWithdrawAndCall(r *runner.E2ERunner, args []string) {
```

Comment:
Some logic could be put in a sub function to perform a withdraw and call with data, we mightn add other test in the future like testing reverts